### ISSUE TITLE
Clarification: OPD vs G-OPD learning rate mismatch (1e-6 vs 1e-5) and fairness of comparison

### ISSUE THREAD

**Issue by liu-s-q19** [RUCBM/G-OPD#11]
Hi authors, thanks for open-sourcing G-OPD.

I noticed that `verl/examples/g_opd/run_qwen3-4b-g-opd.sh` contains two training blocks with different actor learning rates:

* **Standard OPD block** uses:

  ```bash
  actor_rollout_ref.actor.optim.lr=1e-6
  ```

* **G-OPD / ExOPD block** uses:

  ```bash
  actor_rollout_ref.actor.optim.lr=1e-5
  ```

Could you clarify whether this LR difference was intentional in the main reported OPD vs. G-OPD comparisons?

My concern is about fairness and attribution. If OPD and G-OPD use different learning rates by default, some of the observed gains may come from optimization hyperparameters rather than the algorithmic changes themselves.

Do you have a controlled ablation where OPD and G-OPD use the same LR (e.g., both `1e-6` or both `1e-5`) while differing only in the G-OPD-specific components (such as `lambda_vals` and the base/reference correction)?

It would also be very helpful if you could share:

1. The exact hyperparameter table used for the main OPD vs. G-OPD results.
2. Whether the learning rate was tuned separately for each method.
3. Your recommended “fair comparison” setting when reproducing results with this repo.

Thanks!

**Comment by keven980716** [comment]
Hi～Thank you for your attention! In our experiments, the learning rate for both OPD and G-OPD was uniformly set to 1e-5, so the comparison is fair. There is a typo in the script — it is not aligned when we reorganizing the script.

**Comment by liu-s-q19** [comment]
Thank you for your detailed clarification!
